package.json

```
@@ -1,86 +1,34 @@
 {
-  "name": "espree",
-  "description": "An Esprima-compatible JavaScript parser built on Acorn",
-  "author": "Nicholas C. Zakas <nicholas+npm@nczconsulting.com>",
-  "homepage": "https://github.com/eslint/espree",
-  "main": "dist/espree.cjs",
+  "name": "@eslint/js",
+  "version": "1.0.0",
+  "private": true,
   "type": "module",
-  "exports": {
-    ".": [
-      {
-        "import": "./espree.js",
-        "require": "./dist/espree.cjs",
-        "default": "./dist/espree.cjs"
-      },
-      "./dist/espree.cjs"
-    ],
-    "./package.json": "./package.json"
+  "scripts": {
+    "test": "npm test --workspaces --if-present",
+    "build": "npm run build --workspaces --if-present",
+    "build:readme": "node tools/update-readme.js",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ."
   },
-  "version": "10.1.0",
-  "files": [
-    "lib",
-    "dist/espree.cjs",
-    "espree.js"
+  "workspaces": [
+    "packages/*"
   ],
-  "engines": {
-    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
-  },
-  "repository": "eslint/espree",
-  "bugs": {
-    "url": "https://github.com/eslint/espree/issues"
+  "gitHooks": {
+    "pre-commit": "lint-staged"
   },
-  "funding": "https://opencollective.com/eslint",
-  "license": "BSD-2-Clause",
-  "dependencies": {
-    "acorn": "^8.12.0",
-    "acorn-jsx": "^5.3.2",
-    "eslint-visitor-keys": "^4.0.0"
+  "lint-staged": {
+    "*.js": [
```

```suggestion
    "*.{js,cjs}": [
```

To match the previous lint-staged config.

I also think we should now remove `.lintstagedrc.json` as we would have duplicate configs (I _think_ both would apply so eslint would be run twice).